indico/modules/events/management/forms.py

```diff
@@ -232,16 +233,22 @@ def __init__(self, *args, **kwargs):
 
 class EventPrivacyForm(IndicoForm):
     _data_controller_fields = ('data_controller_name', 'data_controller_email')
-    _privacy_policy_fields = ('privacy_policy_url', 'privacy_policy')
+    _privacy_policy_fields = ('privacy_policy_urls', 'privacy_policy')
     data_controller_name = StringField(_('Person/Institution'))
     data_controller_email = EmailField(_('Contact email'), [Optional(), Email()])
-    privacy_policy_url = URLField(_('URL'), [Optional(), URL()],
-                                  description=_('The URL to an external page with the privacy notice'))
+    privacy_policy_urls = MultipleItemsField(_('URLs'),
+                                             fields=[{'id': 'title', 'caption': _('Title'),
+                                                      'required': True, 'type': 'string'},
```

Comment:
It'd be nice if the title was only required in case of multiple privacy notices (especially since we also use i18n as soon as we use a custom title)